docs/guide/exe_verilator.rst

```diff
@@ -129,6 +129,12 @@ Summary:
    is also used). Verilator manages the build itself, and for this --build
    requires GNU Make to be available on the platform.
 
+.. option:: --build-jobs [<value>]
+
+   Specify the level of parallelism for :vlopt:`--build`. The <value> must
+   be a positive integer specifying the maximum number of parallel build
+   jobs.
```

Comment:
```suggestion
   jobs.
   
   See also :vlopt:`-j`.
```

Reply:
Also update docs for -j to say this is default for --build-jobs if --build-jobs isn't provided.